### ISSUE TITLE
Dragon Quest X Offline - Crashes

### ISSUE THREAD

**Issue by nkeeta** [yuzu-emu/yuzu#11452]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access 3855

### Description of Issue

With vulkan the game periodically crashes on loading screens when i changing location and even with autosaves enabled i'm still losing progress. I also installed fan-translation on the game if that helps to solve my problem

### Expected Behavior

The game should go through loading screen without crash

### Reproduction Steps

Start the games
Playing the game
Walk in another location
Loading screen with music but empty loading bar
Crashes

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12522197/yuzu_log.txt)

### System Configuration

CPU: i5 3470
GPU/Driver: 1050ti/536.23
RAM: 12gb
OS: Windows 10

**Comment by Kelebek1** [comment]
Can you run with Graphics Debugging turned on and give another log? It's under the Debug tab in General.

**Comment by SmoothTony** [comment]
Worked on previous version 1560/61 with the English translation mod. Now with translation and without translation it will not load. There is no error message, it appears to simply hang. This is with Vulkan and OpenGL in docked and handheld modes. Using Steam Deck on Windows 10

**Comment by Sn0wCrack** [comment]
Encountering the same issue, doesn't seem to matter if you have DLC, patches or mods applied, seems to black screen pretty much immediately.

Here is my log file with the "Graphics Debugging" setting turned on:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12670883/yuzu_log.txt)

**Comment by username227** [comment]
Yes, a build from last month works fine, but the new version just gives a black screen, both with and without the translation, with both OpenGL and Vulkan.

**Comment by username227** [comment]
FYI, the last build of Yuzu for which DQX works properly is 1559. The bug first appears in 1560. I hope that this helps.

**Comment by Sn0wCrack** [comment]
Seems like latest EA (3971) is working for me again.
